logic/raid_list.php

```diff
@@ -21,7 +25,7 @@ function raid_list($update)
     }
 
     // Inline list polls.
-    if ($iqq != 0) {
+    if ($iqq != 0 or strstr($update['inline_query']['query'],':') == false   ) {
```

Comment:
Better to split this over two lines, i.e. assign the strstr value to a variable that makes it easier to understand what's happening here.